tests/upgradePMM_test.js

```diff
@@ -844,15 +844,14 @@ if (versionMinor >= 13) {
       const {
         serviceType, name, annotationName, dashboard,
       } = current;
-      const timeRange = 'Last 30 minutes';
+      const { service_name } = await inventoryAPI.apiGetNodeInfoByServiceName(serviceType, name, 'ssl');
+      const dashboardUrl = I.buildUrlWithParams(dashboard.split('?')[0], {
+        service_name,
+        from: 'now-30m',
```

Comment:
change this to 5m

Reply:
Are you sure? It may not show up if the upgrade will take more than 5 min

Reply:
I realized we might be getting metrics from before the upgrade (30 mins is big time) if we use that large time offset, instead, we should be fetching the most recent monitored metrics

Reply:
We can chane it to 15 min and see if its stable enough

Reply:
sure go for it